code/modules/mob/living/carbon/human/emote.dm

```diff
@@ -121,6 +121,16 @@
 		return
 	var/mob/living/carbon/human/H = user
 	if(!istype(H) || !H.dna || !H.dna.species || !H.dna.species.can_wag_tail(H))
+		if(prob(0.01))
```

Comment:
1 in 10k is too low

Reply:
should be at least 10%. this is a shitpost, but a good one